apps/server/src/modules/user/legacy/controller/dto/user.response.ts

```diff
@@ -15,6 +15,8 @@ export class UserResponse {
 		consent,
 		classes,
 		importHash,
+		lastLoginSystemChange,
```

Comment:
Why this was removed before?

Reply:
I think they forgot to add it to the response while they moved feathers code to nest.. 
@wiaderwek could you please have a look, was there an important reason ?

Reply:
yeah, it looks like we forgot about that,